docs/embroider-css-modules/app/templates/application.hbs

```diff
@@ -4,7 +4,7 @@
   <header class={{this.styles.header}}>
     <NavigationNarrator @skipTo="#main-content" />
 
-    <NavigationMenu
+    <NavigationMenuNew
```

Comment:
I ran into an oddity when extracting components to `sample-v2-addon`. Namely, I couldn't use the same component name as before.

For the `<NavigationMenu>` component, running the `start` script would result in the following error:

<details>

<summary>Error message</summary>

```sh
[docs-app-for-embroider-css-modules] ERROR in ./templates/application.hbs 2:0-62
[docs-app-for-embroider-css-modules] Module not found: Error: Can't resolve '../components/navigation-menu.ts' in '$TMPDIR/embroider/2fa04b/docs/embroider-css-modules/templates'
[docs-app-for-embroider-css-modules] resolve '../components/navigation-menu.ts' in '$TMPDIR/embroider/2fa04b/docs/embroider-css-modules/templates'
[docs-app-for-embroider-css-modules]   using description file: $TMPDIR/embroider/2fa04b/docs/embroider-css-modules/package.json (relative path: ./templates)
[docs-app-for-embroider-css-modules]     Field 'browser' doesn't contain a valid alias configuration
[docs-app-for-embroider-css-modules]     using description file: $TMPDIR/embroider/2fa04b/docs/embroider-css-modules/package.json (relative path: ./components/navigation-menu.ts)
[docs-app-for-embroider-css-modules]       no extension
[docs-app-for-embroider-css-modules]         Field 'browser' doesn't contain a valid alias configuration
[docs-app-for-embroider-css-modules]         $TMPDIR/embroider/2fa04b/docs/embroider-css-modules/components/navigation-menu.ts doesn't exist
[docs-app-for-embroider-css-modules]       .wasm
[docs-app-for-embroider-css-modules]         Field 'browser' doesn't contain a valid alias configuration
[docs-app-for-embroider-css-modules]         $TMPDIR/embroider/2fa04b/docs/embroider-css-modules/components/navigation-menu.ts.wasm doesn't exist
[docs-app-for-embroider-css-modules]       .mjs
[docs-app-for-embroider-css-modules]         Field 'browser' doesn't contain a valid alias configuration
[docs-app-for-embroider-css-modules]         $TMPDIR/embroider/2fa04b/docs/embroider-css-modules/components/navigation-menu.ts.mjs doesn't exist
[docs-app-for-embroider-css-modules]       .js
[docs-app-for-embroider-css-modules]         Field 'browser' doesn't contain a valid alias configuration
[docs-app-for-embroider-css-modules]         $TMPDIR/embroider/2fa04b/docs/embroider-css-modules/components/navigation-menu.ts.js doesn't exist
[docs-app-for-embroider-css-modules]       .json
[docs-app-for-embroider-css-modules]         Field 'browser' doesn't contain a valid alias configuration
[docs-app-for-embroider-css-modules]         $TMPDIR/embroider/2fa04b/docs/embroider-css-modules/components/navigation-menu.ts.json doesn't exist
[docs-app-for-embroider-css-modules]       .hbs
[docs-app-for-embroider-css-modules]         Field 'browser' doesn't contain a valid alias configuration
[docs-app-for-embroider-css-modules]         $TMPDIR/embroider/2fa04b/docs/embroider-css-modules/components/navigation-menu.ts.hbs doesn't exist
[docs-app-for-embroider-css-modules]       .hbs.js
[docs-app-for-embroider-css-modules]         Field 'browser' doesn't contain a valid alias configuration
[docs-app-for-embroider-css-modules]         $TMPDIR/embroider/2fa04b/docs/embroider-css-modules/components/navigation-menu.ts.hbs.js doesn't exist
[docs-app-for-embroider-css-modules]       .ts
[docs-app-for-embroider-css-modules]         Field 'browser' doesn't contain a valid alias configuration
[docs-app-for-embroider-css-modules]         $TMPDIR/embroider/2fa04b/docs/embroider-css-modules/components/navigation-menu.ts.ts doesn't exist
[docs-app-for-embroider-css-modules]       as directory
[docs-app-for-embroider-css-modules]         $TMPDIR/embroider/2fa04b/docs/embroider-css-modules/components/navigation-menu.ts doesn't exist
[docs-app-for-embroider-css-modules]  @ ./assets/docs-app-for-embroider-css-modules.js 56:13-52
[docs-app-for-embroider-css-modules] 
[docs-app-for-embroider-css-modules] webpack 5.81.0 compiled with 1 error in 4979 ms
[docs-app-for-embroider-css-modules] Build Error (PackagerRunner) in templates/application.hbs
[docs-app-for-embroider-css-modules] 
[docs-app-for-embroider-css-modules] Module not found: Error: Can't resolve '../components/navigation-menu.ts' in '$TMPDIR/embroider/2fa04b/docs/embroider-css-modules/templates/application.hbs'
```

</details>

I suspect that the error may be fixed by installing newer versions of `@embroider/*` packages.